recipes-dom0/dom0-image-weston/files/meta-xt-extra/recipes-extended/guest-addons/guest-addons.bb

```diff
@@ -0,0 +1,33 @@
+# Copyright
+# License:
+#
+# Filename: guest-addons.bb
```

Comment:
no, this is not only IOMMU tests, but all the files needed for Dom0 to serve guests.
Thus, to me all the configuration files, tests etc is a part of guest-addons package.
What we can probably do is split this package into number of more specialized ones,
but does it make any sense ATM?